src/environments/environment.prod.ts

```diff
@@ -1,4 +1,4 @@
 export const environment = {
   production: true,
-  apiURL: 'http://18.233.7.153:9001',
+  apiURL: 'http://cashoverflow2-env.eba-vs9b3n6r.us-east-1.elasticbeanstalk.com:5000',
```

Comment:
@mbaileyfuturist my bad, please remove the port number! It's automatically included - so it should just look like  `'http://cashoverflow2-env.eba-vs9b3n6r.us-east-1.elasticbeanstalk.com'`